src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write your code here
+const allSpans = document.querySelectorAll('.population');
+const totalSpan = document.querySelector('.total-population');
+const averageSpan = document.querySelector('.average-population');
+let totalValue = 0;
+
+for (let i = 0; i < allSpans.length; i++) {
+  let arr = [];
+
+  arr = allSpans[i].textContent.split(',');
+
+  const value = arr.join('');
+
+  totalValue += +value;
+}
+
+const averageValue = Math.ceil(totalValue / allSpans.length);
+
+const totalValueStr = totalValue.toString().split('');
+const averageValueStr = ('' + averageValue).split('');
+
+const resTotal = [
+  ...totalValueStr.slice(0, 1),
+  ',',
+  ...totalValueStr.slice(1, 4),
+  ',',
+  ...totalValueStr.slice(4, -3),
+  ',',
+  ...totalValueStr.slice(-3),
+];
+
+const resAverage = [
+  ...averageValueStr.slice(0, 3),
+  ',',
+  ...averageValueStr.slice(3, 6),
+  ',',
+  ...averageValueStr.slice(6),
+];
```

Comment:
Similar to the `totalValue` formatting, the logic for `averageValue` assumes a fixed number of digits. This can lead to incorrect formatting if the number of digits is less than expected. Consider using a more flexible approach to format numbers with commas.